spec/models/comment_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+describe Comment do
+  describe "table" do
+    it { should have_db_column(:rating).of_type(:integer) }
+    it { should have_db_column(:body).of_type(:string) }
+    it { should have_db_column(:created_at).of_type(:datetime) }
+    it { should have_db_column(:updated_at).of_type(:datetime) }
+  end
+
+  describe "associations" do
+    it { should belong_to(:user) }
+    it { should belong_to(:move) }
+  end
+
+  describe "validations" do
+    it { should have_valid(:rating).when (5) }
+    it { should validate_inclusion_of(:rating).in_range(1..5).
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.